Dockerfile

```diff
@@ -1,46 +1,95 @@
-FROM        registry.access.redhat.com/ubi9/python-39:9.5-1739799514@sha256:9bbc4cfeac896544ab3eafa088c3d6995e82592362d374606f00d221f2986fe0 as prod
+##############
+# base image #
+##############
+FROM registry.access.redhat.com/ubi9/python-39:9.5-1739799514@sha256:9bbc4cfeac896544ab3eafa088c3d6995e82592362d374606f00d221f2986fe0 as base
 
+COPY        LICENSE /licenses/LICENSE
+
+#################
+# builder image #
+#################
+FROM base as builder
```

Comment:
fix warning `WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match`

use 

```dockerfile
FROM base AS builder
```